packages/pwa-kit-react-sdk/CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+## v1.3.0 (Jan 06, 2022)
```

Comment:
Hmm.. there doesn't seem to be other changelog files. I expect to see the other ones updated too.